README.md

```diff
@@ -85,7 +85,7 @@ var elements = document.querySelectorAll('.editable'),
 
 ### Toolbar options
 * __activeButtonClass__: CSS class added to active buttons in the toolbar. Default: 'medium-editor-button-active'
-* __buttons__: the set of buttons to display on the toolbar. Default: ['bold', 'italic', 'underline', 'anchor', 'header1', 'header2', 'quote']
+* __buttons__: the set of buttons to display on the toolbar. Default: `['bold', 'italic', 'underline', 'fontsize', 'anchor', 'header1', 'header2', 'quote']`
```

Comment:
In the spirit of having this look like medium by default, we probably should have this as a default button.

There is a section for listing additional buttons further down in the documentation, so you should definitely add this into that list.